sensors/github_repository_sensor.yaml

```
@@ -1,24 +1,24 @@
 ---
-  class_name: "GithubRepositorySensor"
-  entry_point: "github_repository_sensor.py"
-  description: "Sensor which monitors Github repository for activity"
-  # requires authentication because of GitHub rate limiting
-  # use default with auth or increase poll interval
-  poll_interval: 30
-  trigger_types:
-    -
-      name: "repository_event"
-      description: "Trigger which indicates a new repository action"
-      payload_schema:
-        type: "object"
-        properties:
-          id:
-            type: "string"
-          created_at:
-            type: "string"
-          type:
-            type: "string"
-          actor:
-            type: "object"
-          payload:
-            type: "object"
+class_name: "GithubRepositorySensor"
+entry_point: "github_repository_sensor.py"
+description: "Sensor which monitors Github repository for activity"
+# requires authentication because of GitHub rate limiting
+# use default with auth or increase poll interval
+enabled: false
```

This sensor was enabled by default. Let's please keep it enabled.
```suggestion
enabled: true
```